apps/ledger-live-desktop/src/renderer/families/evm/SendAmountFields/SelectFeeStrategy.tsx

```diff
@@ -0,0 +1,163 @@
+import { getEstimatedFees } from "@ledgerhq/coin-evm/logic";
+import { getTypedTransaction } from "@ledgerhq/coin-evm/transaction";
+import { Transaction as EvmTransaction, GasOptions } from "@ledgerhq/coin-evm/types";
+import { getAccountCurrency, getAccountUnit } from "@ledgerhq/live-common/account/index";
+import { Account } from "@ledgerhq/types-live";
+import React, { memo, useMemo } from "react";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import Box, { Tabbable } from "~/renderer/components/Box";
+import CounterValue from "~/renderer/components/CounterValue";
+import FormattedVal from "~/renderer/components/FormattedVal";
+import Text from "~/renderer/components/Text";
+import Clock from "~/renderer/icons/Clock";
+import TachometerHigh from "~/renderer/icons/TachometerHigh";
+import TachometerLow from "~/renderer/icons/TachometerLow";
+import TachometerMedium from "~/renderer/icons/TachometerMedium";
+
+type OnClickType = {
+  feesStrategy: keyof GasOptions;
+};
+
+type Props = {
+  onClick: (arg: OnClickType) => void;
```

Comment:
```suggestion
  onClick: (arg: keyof GasOptions) => void;
```

Are these two options the same ? If yes could you go with the latter please ?

Reply:
Done here 9c3313126b343e8815c7c7733aa81e528824df1f